src/Components/People.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import axios from 'axios'
+export class People extends Component {
+constructor() {
+    super();
+    this.state = {
+        searchPerson: "",
+        currPerson: {},
+        currName: '',
+        isError: false,
+    }
+}
+
+handleChange = (e) => {
+    this.setState({
+        searchPerson: e.target.value,
+    })
+}
+
+handleSubmit= async (e) => {
+    e.preventDefault();
+    const { searchPerson } = this.state
+
+    try {
+        const { data } = await axios.get(
+          `https://ghibliapi.herokuapp.com/people`
+        );
+            const person = data.find((ele) => ele.name === searchPerson)
+        this.setState({
+          currPerson: person,
+          currName: person.name,
+          searchPerson: "",
+          isError: false,
+        });
+      } catch (e) {
+        this.setState({
+          currPerson: {},
+          currName:"",
+          searchPerson: "",
+          isError: true,
+        });
+      }
+    };
+
+    render() {
+        const { searchPerson, currPerson, currName, isError } = this.state
+        return (
+            <div style={{
+                backgroundColor: 'rgb(128, 59, 59)',
+                width: 'auto',
+                height: '500px'
+              }}
+            >
+                <h1>Search for a Person</h1>
+                <form onSubmit={this.handleSubmit}>
+                    <input 
+                    value={searchPerson}
+                    onChange={this.handleChange} 
+                    type="text"
+                     placeholder="Find Your Person"
+                     />
+                    <button>Submit</button>
+                </form>
+                {currName ? (
```

Comment:
You can also write this line:
{currName ? (

) : null}
as:
{currName && (

)}